packages/shared/src/utils/systemTimeUtils.ts

```
@@ -0,0 +1,104 @@
+import axios from 'axios';
+import { isNaN, isNil } from 'lodash';
```

Biome (1.9.4)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)